react/__tests__/Telemarketing.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { render } from 'react-testing-library'
+
+import Telemarketing from '../components/Telemarketing'
+import messages from '../../messages/en-US.json'
+
+describe('<Telemarketing /> component', () => {
+  const intl = {
```

Comment:
It will be nice to use `IntlPRovider`

Reply:
`IntlProvider` does not work with this component, because It uses `injectIntl`. I tryied to use `injectIntl` but the snapshot created was pretty bad, just with the `<DocumentFragment>`.